precompiles/erc20/approve.go

```diff
@@ -43,11 +43,7 @@ func (p Precompile) Approve(
 	granter := contract.CallerAddress
 
 	// TODO: owner should be the owner of the contract
-	authorization, _, err := auth.CheckAuthzExists(ctx, p.AuthzKeeper, grantee, granter, SendMsgURL)
-	if err != nil {
-		return nil, err
-	}
-	// case 1: authorization doesn't exist
```

Comment:
if there is no authorization, this function returns an error - however, we expect the situation where no prior approval exists right? Hence, this check for the error needs to be removed

Reply:
good catch